docs/source/using.rst

```diff
@@ -80,7 +80,8 @@ suite are:
 - ``environment_type``: The tool used to create environments.  May be
   ``conda`` or ``virtualenv`` or ``mamba``.  If Conda supports the
   dependencies you need, that is the recommended method. Mamba is faster
-  but needs a newer Python version (3.8 or greater).
+  but needs a newer Python version (3.8 or greater) and the ``libmambapy``
+  package must be installed to provide access to the mamba Python API.
```

Comment:
```suggestion
  but needs a newer Python version (3.8 or greater) and the ``libmambapy``
  package must be installed (typically through ``conda``) to provide access to the mamba Python API.
```

Clarification since `libmambapy` is not present on PyPI.